docs/developers/document-releasing.md

```diff
@@ -40,13 +40,17 @@ go run ./hack/tools/genkarmadactldocs/gen_karmadactl_docs.go ../website/docs/ref
 go run ./hack/tools/genkarmadactldocs/gen_karmadactl_docs.go ../website/i18n/zh/docusaurus-plugin-content-docs/current/reference/karmadactl/karmadactl-commands/
 ```
 
-3. Generate reference docs of each component one by one. Here we take `karmada-apiserver` as an example.
+3. Generate reference docs of each component.
 
 ```shell
 cd karmada/
 go build ./hack/tools/gencomponentdocs/.
-./gencomponentdocs ../website/docs/reference/components/ karmada-apiserver
-./gencomponentdocs ../website/i18n/zh/docusaurus-plugin-content-docs/current/reference/components/ karmada-apiserver
+
+mods=(karmada-controller-manager karmada-scheduler karmada-agent karmada-aggregated-apiserver karmada-descheduler karmada-search karmada-scheduler-estimator karmada-webhook karmada-metrics-adapter)
+for mod in ${mods};do 
```

Comment:
```suggestion
for component in ${components};do 
```
We don't usually say modules.

Reply:
Not necessary. That code was inherited from Kubernetes.

But in doc, we say components.